postgrest.cabal

```diff
@@ -203,22 +204,23 @@ test-suite spec
                       Feature.Query.DeleteSpec
                       Feature.Query.EmbedDisambiguationSpec
                       Feature.Query.EmbedInnerJoinSpec
-                      Feature.Query.PlanSpec
+                      Feature.Query.ErrorSpec
                       Feature.Query.HtmlRawOutputSpec
                       Feature.Query.InsertSpec
                       Feature.Query.JsonOperatorSpec
                       Feature.Query.MultipleSchemaSpec
-                      Feature.Query.ErrorSpec
+                      Feature.Query.NullsStrip
```

Comment:
If `NullsStrip` mutates the db (not sure), then this change of order might also affect the idempotence test.

Reply:
I don't think the order matters here since, `postgrest.cabal` simply lists the build and test dependencies.

Reply:
Ah my bad, I thought this list was somewhere in the SpecHelper.